src/sentry/search/events/datasets/function_aliases.py

```diff
@@ -11,16 +11,19 @@
 from sentry.search.events import constants
 from sentry.search.events.types import SelectType
 from sentry.sentry_metrics.configuration import UseCaseKey
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
 
 
 def resolve_project_threshold_config(
```

Comment:
The only caller to this function can be found here: https://github.com/getsentry/sentry/blob/cf2dd1348ee1e8bf9654972705b986815306637c/src/sentry/snuba/metrics/fields/snql.py#L735-L745